cockroachdb/templates/statefulset.yaml

```diff
@@ -128,6 +128,21 @@ spec:
         {{- end }}
       {{- end }}
     {{- end }}
+    {{- if semverCompare ">=1.16-0" .Capabilities.KubeVersion.Version }}
```

Comment:
I don't think this check is working as expected, based on the [test failure](https://teamcity.cockroachdb.com/viewLog.html?buildId=2326789&buildTypeId=Cockroach_HelmCharts_ChartsGitHubCi&tab=buildLog&_focus=465). `.Capabilities.KubeVersion.Version` [appears](https://github.com/helm/helm/blob/0ca95341c19fcaf9cecdf50caef1693568835a84/pkg/chartutil/capabilities.go#L48) to be a [k8s.io/apimachinery/pkg/version#Info](https://godoc.org/k8s.io/apimachinery/pkg/version#Info), which does not have a `Version` field.

Reply:
changed to `.Capabilities.KubeVersion.GitVersion` which is deprecated on Helm v3 but still works :-)